lib/explorer/series.ex

```diff
@@ -1864,6 +1864,53 @@ defmodule Explorer.Series do
   def quantile(%Series{dtype: dtype}, _),
     do: dtype_error("quantile/2", dtype, [:integer, :float, :date, :time, :datetime])
 
+  @doc """
+  Compute the sample skewness of a data set.
+
+  For normally distributed data, the skewness should be about zero. For unimodal continuous distributions, a skewness value greater than zero means that there is more weight in the right tail of the distribution. The function skewtest can be used to determine if the skewness value is close enough to zero, statistically speaking.
+
+  See scipy.stats for more information.
```

Comment:
We should not point to scipy :D

Reply:
haha my bad, copied the description from polars and forgot to edit it. I'll change it and the ref to skewtest.